presto-parquet/src/main/java/com/facebook/presto/parquet/dictionary/DoubleDictionary.java

```diff
@@ -31,7 +33,8 @@ public DoubleDictionary(DictionaryPage dictionaryPage)
         super(dictionaryPage.getEncoding());
         content = new double[dictionaryPage.getDictionarySize()];
         DoublePlainValuesReader doubleReader = new DoublePlainValuesReader();
-        doubleReader.initFromPage(dictionaryPage.getDictionarySize(), dictionaryPage.getSlice().getBytes(), 0);
+        ByteBufferInputStream inputStream = ByteBufferInputStream.wrap(ByteBuffer.wrap(dictionaryPage.getSlice().getBytes()));
```

Comment:
I don't know how hot this method will be, but the `ByteBufferInputStream.wrap` call is a var args call, which is slow in general. We can do:

```
ByteBuffer byteBuffer = ByteBuffer.wrap(dictionaryPage.getSlice().getBytes());
ByteBufferInputStream inputStream = ByteBufferInputStream.wrap(ImmutableList.of(byteBuffer));
```

Reply:
Done.